package.json

```diff
@@ -129,6 +129,8 @@
     "electron:packager-win": "electron-packager .package bitburner --platform win32 --arch x64 --out .build --overwrite --icon .package/icon.png",
     "electron:packager-mac": "electron-packager .package bitburner --platform darwin --arch x64 --out .build --overwrite --icon .package/icon.png",
     "electron:packager-linux": "electron-packager .package bitburner --platform linux --arch x64 --out .build --overwrite --icon .package/icon.png",
-    "allbuild": "npm run build && npm run electron && git add --all && git commit -m \"allbuild commit $(which git)\" && git push -f -u origin dev"
+    "preversion": "npm install && npm run test",
+    "version": "sh ./tools/build-release.sh && git add --all",
```

Comment:
Why not something like `build` or even keeping the old name? I'd expect `version` to print the current version or something like that

Reply:
`npm version` is a built in npm command with its preversion & postversion hooks. See the [npmjs doc](https://docs.npmjs.com/cli/v6/commands/npm-version). If we'd keep `allbuild` or `build` we would need to forward command's args.

Reply:
You learn something new every day